docker/pytorch-aarch64/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ where `YY` is the year, and `MM` the month of the increment.
 
 ### Added
 - Adds torchdata build script
+- Adds torch xla build option to the pytorch-dev image
```

Comment:
it's not clear from this comment that the XLA build option applies _from_ the dev image onwards, i.e. it will impact the full build too.

Reply:
yes, any layer from pytorch-dev onwards, I will update the note.